docs/technical-documentation/checking-coding-standards.md

```diff
@@ -1,16 +1,15 @@
 
 # Checking Code Style
 
-Before opening a pull request, you should check your code style. If you are using the [Vagrant](https://github.com/Islandora-Devops/claw-playbook), you can run `phpcs` within the Drupal installation directory (on the Vagrant, that is `/var/www/html/drupal`) or from within the `web` directory (`/var/www/html/drupal/web`) as follows:
+Before opening a pull request, you should check your code style using PHP_CodeSniffer (phpcs). We use the Drupal coding standards provided by [Drupal coder](https://www.drupal.org/project/coder). If you are using the [Ansible Playbook](https://github.com/Islandora-Devops/islandora-playbook), phpcs and Drupal Coder are installed and phpcs can be run as follows:
 
-* from within Drupal's root directory: `./vendor/bin/phpcs --standard=./vendor/drupal/coder/coder_sniffer/Drupal modules/contrib/my_module`, where `modules/contrib/my_module` is the relative or full path to the PHP file you want to check.
-* from within Drupal's `web` directory: `../vendor/bin/phpcs --standard=../vendor/drupal/coder/coder_sniffer/Drupal yourfile`, where `yourfile` is the relative or full path to the PHP file you want to check.
+* from within Drupal's root directory: `./vendor/bin/phpcs --standard=./vendor/drupal/coder/coder_sniffer/Drupal --ignore=*.md --extensions=php,module,inc,install,test,profile,theme,css,info web/modules/contrib/my_module`
+* from within Drupal's `web` directory: `../vendor/bin/phpcs --standard=../vendor/drupal/coder/coder_sniffer/Drupal --ignore=*.md --extensions=php,module,inc,install,test,profile,theme,css,info modules/contrib/my_module`
```

Comment:
we also use the `DrupalPractice` library

Reply:
@elizoller I thought i knew what you meant but I'm not sure that I do!

In https://github.com/Islandora/islandora_ci/blob/main/travis_scripts.sh we run two commands: `phpcs --standard=Drupal` and `phpcpd`.  The second one looks to me to be a "copy paste detector". 

Looking into Drupal's docs on [PHP CodeSniffer CommandLine Usage](https://www.drupal.org/node/1587138), they mention "Check Drupal Best Practices" with the example `phpcs --standard=DrupalPractice`.

I don't see where we actually run this in our code checks. From my knowledge, what's in travis_scripts.sh in the islandora_ci library is the code check that applies to all our repos.

@dannylamb or @whikloj might also know?